### PR TITLE
Doc fixes for PolylineGraphics

### PR DESCRIPTION
**Pull request by denverpierce**
Fixes missing doc strings for #5886

### CODE REVIEW

**Comment by cesium-concierge** [comment]
@denverpierce, thanks for the pull request! Maintainers, we have a signed CLA from @denverpierce, so you can review this at any time.

I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).



__I am a bot who helps you make Cesium awesome!__ Thanks again.

File: Source/DataSources/PolylineGraphics.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ define([
      * @param {Property} [options.width=1.0] A numeric Property specifying the width in pixels.
      * @param {Property} [options.show=true] A boolean Property specifying the visibility of the polyline.
      * @param {MaterialProperty} [options.material=Color.WHITE] A Property specifying the material used to draw the polyline.
+     * @param {MaterialProperty} [options.depthFailMaterial=undefinded] A property specifiying the material to used to draw the polyline when it is below the terrain.
```
Comment (hpinkos) [inline Source/DataSources/PolylineGraphics.js]: We usually just do `[options.depthFailMaterial]` instead of `[options.depthFailMaterial=undefinded]`

**Comment by hpinkos** [comment]
Thanks for catching those @denverpierce!  I'll merge this once you fix that one comment I had

**Comment by denverpierce** [comment]
Fixed.

**Comment by hpinkos** [comment]
👍 👍

**Pull request merged**